packages/forms/resources/views/components/table-builder.blade.php

```diff
@@ -0,0 +1,123 @@
+                            ]) scope="col">
+                                <div class="flex items-center justify-between">
+                                    <span x-text="i"></span>
+
+                                    @if($canDeleteColumns() && ! $isDisabled())
+                                        <button
+                                            x-on:click="removeColumn(i)"
+                                            x-show="columns > 1"
+                                            type="button"
+                                            class="text-gray-400 hover:text-gray-600 focus:text-gray-600"
+                                        >
+                                            <x-heroicon-o-x class="w-4 h-4" />
+                                        </button>
+                                    @endif
+                                </div>
+                            </th>
+                        </template>
+
+                        @if($deleteRows && ! $isDisabled())
+                            <th x-show="state.length > 1" class="w-10">
+
+                            </th>
+                        @endif
+                    </tr>
+                </thead>
+
+                <tbody @class([
+                    'divide-y whitespace-nowrap',
+                    'dark:divide-gray-600' => config('forms.dark_mode'),
+                ])>
+                    <template x-for="(row, index) in state" x-bind:key="index">
+                        <tr @class([
+                            'divide-x',
+                            'dark:divide-gray-600' => config('forms.dark_mode'),
+                        ])>
+                            <template x-for="column in columns">
+                                <td>
+                                    <input
+                                        type="text"
+                                        x-model="row[column - 1]"
+                                        @if ($isDisabled())
+                                            disabled
+                                        @endif
+                                        class="w-full px-4 py-3 font-mono text-sm bg-transparent border-0 focus:ring-0"
+                                    >
+                                </td>
+                            </template>
+
+                            @if($deleteRows && ! $isDisabled())
+                                <td x-show="state.length > 1" class="p-0">
+                                    <div class="flex items-center justify-center">
+                                        <button
+                                            x-on:click="removeRow(index)"
+                                            type="button"
+                                            class="text-gray-400 hover:text-gray-600 focus:text-gray-600"
+                                        >
+                                            <x-heroicon-o-x class="w-4 h-4" />
```

Comment:
What if we use this?
```suggestion
                                            <x-heroicon-s-x class="w-5 h-5" />
```